<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="GraphicsLayoutWidget" name="graph"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="zeroOffset">
       <property name="text">
        <string>Zero offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bwOverride">
       <property name="text">
        <string>Bandwidth factor override</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="bwFactor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trackEnable">
       <property name="toolTip">
        <string>Enable a tracking algorithm to automatically adjust the frequency and span to follow the resonant peak</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Enable automatic tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forceRetrack">
       <property name="text">
        <string>Force retrack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetSegments">
       <property name="toolTip">
        <string>Reset segments to their configuration values</string>
       </property>
       <property name="text">
        <string>Reset segments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="segmentTable"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Duration to display (hh:mm:ss):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="historicalDuration">
       <property name="minimumTime">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>1</second>
        </time>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>5</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bwOverride</sender>
   <signal>clicked(bool)</signal>
   <receiver>bwFactor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
